### PR TITLE
Add pr checker for ut

### PR DESCRIPTION
**Pull request by chalsliu**
Get the list of PR authors with unresolved unit test issues.

### CODE REVIEW

**Review by kolinwei — DISMISSED** [review]

File: tools/check_api_approvals.sh [hunk 1]
```diff
@@ -283,6 +283,14 @@ if [ "${ADDED_OP_USE_DEFAULT_GRAD_MAKER}" != "" ]; then
   check_approval 1 32832641 6836917
 fi
 
+pip install PyGithub 
+wget https://paddle-ci.gz.bcebos.com/blk/block.txt
+HASUTFIXED=`python ${PADDLE_ROOT}/tools/check_ut.py | grep "has UT to be fixed" || true`
+if [ "${HASUTFIXED}" != "" ]; then
+    echo_line="${HASUTFIXED} You must have one RD (kolinwei or zhouwei25) approval.\n"
```
Comment (luotao1) [inline tools/check_api_approvals.sh]: - zhouwei25不太适合吧，换一个人？
- 请参考其他，需要有一个推荐的，即类似kolinwei(recommended) or zhouwei25

File: tools/check_api_approvals.sh [hunk 2]
```diff
@@ -283,6 +283,14 @@ if [ "${ADDED_OP_USE_DEFAULT_GRAD_MAKER}" != "" ]; then
   check_approval 1 32832641 6836917
 fi
 
+pip install PyGithub
```
Comment (luotao1) [inline tools/check_api_approvals.sh]: 需要加一下注释说明这段的作用

File: tools/check_api_approvals.sh [hunk 3]
```diff
@@ -283,6 +283,15 @@ if [ "${ADDED_OP_USE_DEFAULT_GRAD_MAKER}" != "" ]; then
   check_approval 1 32832641 6836917
 fi
 
+# For getting PR related data
```
Comment (luotao1) [inline tools/check_api_approvals.sh]: 需要加一下整体这段逻辑的注释，目前286行是PyGithub的注释

File: tools/check_api_approvals.sh [hunk 4]
```diff
@@ -283,6 +283,15 @@ if [ "${ADDED_OP_USE_DEFAULT_GRAD_MAKER}" != "" ]; then
...
+pip install PyGithub
+wget https://paddle-ci.gz.bcebos.com/blk/block.txt
+HASUTFIXED=`python ${PADDLE_ROOT}/tools/check_ut.py | grep "has UT to be fixed" || true`
```
Comment (luotao1) [inline tools/check_api_approvals.sh]: has unit-test to be fixed

**Comment by chalsliu** [comment]
Get the list of PR authors with unresolved unit test issues.

File: tools/check_api_approvals.sh (hunk 1 above)
Comment (chalsliu) [reply tools/check_api_approvals.sh]: done

File: tools/check_api_approvals.sh (hunk 2 above)
Comment (chalsliu) [reply tools/check_api_approvals.sh]: done

File: tools/check_api_approvals.sh (hunk 3 above)
Comment (chalsliu) [reply tools/check_api_approvals.sh]: done

File: tools/check_api_approvals.sh (hunk 4 above)
Comment (chalsliu) [reply tools/check_api_approvals.sh]: done

**Review by luotao1 — APPROVED** [review]
LGTM

**Pull request merged**